base/reduce.jl

```diff
@@ -900,6 +900,8 @@ Values are compared with `isless`.
 !!! compat "Julia 1.7"
     This method requires Julia 1.7 or later.
 
+See also: [`argmin`](@ref), [`findmax`](@ref).
```

Comment:
```suggestion
See also [`argmin`](@ref), [`findmax`](@ref).
```

Reply:
Seems like that PR was not consistently applied across Base, I've just followed the convention in that file:

```
julia $ rg "See also\s" base/ | wc -l          
230                                                            
julia $ rg "See also:\s" base/ | wc -l         
131                                                            
julia $ rg "See also\s" base/reduce.jl | wc -l 
0                                                              
julia $ rg "See also:\s" base/reduce.jl | wc -l
11                                                             
```

Should I change it in all of reduce.jl? There are some erronous `,` in other docstrings as well, looking through them.

Reply:
Hmm, maybe those were merged after that PR then. Could do a follow up PR to fix the rest, but no need to introduce more of them.

Reply:
I don't know, it just seems like a lot of `See also`s - is there some hook in Documenter.jl or some doc CI where such things could be checked?

Reply:
No it is just a convention (as documented in the manual).